### PR TITLE
Add instructions for disabling security check

### LINKED ISSUE #16: cask install/upgrade leaves `com.apple.quarantine` on GraalVM, blocking execution
After installation or upgrade of the GraalVM 11 cask (multiple versions, 19 and 20) the `/Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.0.0` directory and content have the `com.apple.quarantine` xattr present.

This is the "normal" behaviour of downloaded software on MacOS, but it causes problems running from the CLI: it pops up a dialog box informing me that the software can't be run, and allows no override.

To resolve this it is sufficient to execute this command:
```shell
xattr -d -r com.apple.quarantine /Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.0.0
```

I'd suggest updating the cask to automatically apply that to the appropriate directory after unpacking and moving.  That'd strip off the quarantine flag, allowing execution of the content.

Normally I wouldn't encourage automatically removing an OS security flag, but in this case it seems justified: I explicitly requested the software be installed via brew cask, so I'd expect it to be flagged as "safe" to use immediately afterwards.

### PR DESCRIPTION
**Pull request by fniephaus**
Closes #16

### CODE REVIEW

File: Casks/graalvm-ce-java11.rb [hunk 1]
```diff
@@ -29,6 +29,11 @@
     Installing GraalVM CE (Java 11) in #{JVMS_DIR} requires root permissions.
     You may be asked to enter your password to proceed.
 
+    The first time you run GraalVM CE on macOS Catalina, you may get a warning
```
Comment (dwijnand) [inline Casks/graalvm-ce-java11.rb]: It's also on upgrades (`brew cask upgrade`), or at least the `xattr` way requires running it multiple times (does clicking the button run `xattr`? 🤷‍♂).

File: Casks/graalvm-ce-java11.rb [hunk 2]
```diff
@@ -29,6 +29,11 @@
...
+    that "the developer cannot be verified". This check can be disabled in the
+    "Security & Privacy" preferences pane or by running the following command:
+      xattr -r -d com.apple.quarantine #{TARGET_DIR}
```
Comment (dwijnand) [inline Casks/graalvm-ce-java11.rb]:
```suggestion
      xattr -d com.apple.quarantine #{TARGET_DIR}
```

Comment (fniephaus) [reply Casks/graalvm-ce-java11.rb]: Are you sure this does not have to run recursively?

Comment (fniephaus) [reply Casks/graalvm-ce-java11.rb]: GraalVM isn't following the normal app bundle layout, so I'm for keeping `-r`.

Comment (dwijnand) [reply Casks/graalvm-ce-java11.rb]: Following your advice in https://github.com/graalvm/homebrew-tap/issues/6#issuecomment-539474120, I've successfully resolved the issue back when I installed 19.3 and when I recently upgraded to 20.0, without needing the `-r`.  So it seems like a better, smaller hammer.  Then again, it's just informational here...

Comment (fniephaus) [reply Casks/graalvm-ce-java11.rb]: If we are sure `-r` is definitely not needed, happy to remove it. But I'd probably keep it for now. Hopefully the release bundles will be properly signed sometime soon, so this all will go away.

**Comment by fniephaus** [comment]
@dougxc could you merge this please?

**Pull request merged**